### PR TITLE
Work around name clash of bitfield and unit fields on spicy-1.9

### PR DESCRIPTION
**Pull request by bbannier**

### CODE REVIEW

File: analyzer/asn1.spicy [hunk 1]
```diff
@@ -92,7 +92,8 @@ type ASN1Tag = unit {
   var class: ASN1Class;
   var constructed: bool;
 
-  : bitfield(8) {
+  # TODO(bbannier): Use an anonymous bitfield here once zeek/spicy#1533 is fixed.
```
Comment (rsmmr) [inline analyzer/asn1.spicy]: This isn't related to #1533, is it?

**Review by rsmmr — APPROVED** [review]

File: analyzer/asn1.spicy (hunk 1 above)
Comment (bbannier) [reply analyzer/asn1.spicy]: It is indirectly. With spicy-1.9 I would want to use an anonymous bitfield again and remove the unit vars. This then fails when an `ASN1Tag` is used elsewhere (here: passed as a unit parameter exactly as in zeek/spicy#1533).

**Comment by bbannier** [comment]
I'll wait with merging this. With zeek/spicy#1537 we have a PR open to fix the mentioned issue and might want to wait for this to land in zeek/spicy and zeek/zeek.

**Comment by bbannier** [comment]
> I'll wait with merging this. With [zeek/spicy#1537](https://github.com/zeek/spicy/pull/1537) we have a PR open to fix the mentioned issue and might want to wait for this to land in zeek/spicy and zeek/zeek.

The analyzer has been reworked to use the modern form on >=spicy-1.9.0.

**Review by rsmmr — APPROVED** [review]

**Pull request merged**